### PR TITLE
misc pn updates

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: src/components/map-select-tools/map-select-tools.tsx [hunk 1]
```diff
@@ -512,7 +512,7 @@ export class MapSelectTools {
               {this._translations.searchDistance}
               <calcite-icon
                 class="padding-start-1-2 icon"
-                flipRtl={!(this.locale.toLowerCase() === "he")}
+                flipRtl={!(this.locale?.toLowerCase() === "he")}
```
Comment (MikeTschudi) [inline src/components/map-select-tools/map-select-tools.tsx:515]: Shouldn't the 5 rtl checks be
```ts
flipRtl=true
```
(TypeScript style)
or
```html
flip-rtl
```
(HTML style)?

The [flipRtl](https://developers.arcgis.com/calcite-design-system/components/icon/#api-reference-properties-flipRtl) doc says, "When `true`, the icon will be flipped when the element direction is right-to-left (`"rtl"`).", which should have been set for the whole page.

Comment (jmhauck) [reply src/components/map-select-tools/map-select-tools.tsx:515]: @MikeTschudi no its a specific case for HE `?` are not supposed to be flipped. 

image


https://esriis.sharepoint.com/sites/i18n_t9n/SitePages/How-to-test-Arabic-and-Hebrew-languages.aspx

**Comment by jmhauck** [comment]
#790 component logic added still need to handle from IA config

File: src/components/map-select-tools/map-select-tools.tsx (hunk 1 above)
Comment (MikeTschudi) [reply src/components/map-select-tools/map-select-tools.tsx:533]: Thank you; I remember it now.

**Review by MikeTschudi — APPROVED** [review]

**Pull request merged**